.cargo/config.toml

```diff
@@ -5,6 +5,7 @@ CLI_VERSION_OVERRIDE = "v1.3.1-priority.0"
 
 [alias]
 integ-test = ["run", "--package", "temporal-sdk-core", "--example", "integ_runner", "--"]
+cloud-test = ["run", "--package", "temporal-sdk-core", "--example", "integ_runner", "--", "--test-name", "cloud_tests"]
```

Comment:
You can bypass the integ runner here (and it saves some compile time) since all that's really doing is starting the local server, which ofc you don't need for cloud.